simd/src/CMakeLists.txt

```
@@ -30,6 +30,16 @@ kokkos_add_library(
   MODULE_INTERFACE
   ${SIMD_MODULES}
 )
+
+if(KOKKOS_CXX_COMPILER_ID STREQUAL Intel OR KOKKOS_CXX_COMPILER_ID STREQUAL IntelLLVM)
```

We don't support Intel classic anymore
```suggestion
if(KOKKOS_CXX_COMPILER_ID STREQUAL IntelLLVM)
```